clamav.py

```diff
@@ -60,51 +69,100 @@ def update_defs_from_s3(s3_client, bucket, prefix):
             s3_time = time_from_s3(s3_client, bucket, s3_path)
 
             if s3_best_time is not None and s3_time < s3_best_time:
-                print("Not downloading older file in series: %s" % filename)
+                older_files.add(filename)
                 continue
             else:
                 s3_best_time = s3_time
 
             if os.path.exists(local_path) and md5_from_file(local_path) == s3_md5:
-                print("Not downloading %s because local md5 matches s3." % filename)
+                md5_matches.add(filename)
                 continue
             if s3_md5:
                 to_download[file_prefix] = {
                     "s3_path": s3_path,
                     "local_path": local_path,
                 }
+
+    if AV_EXTRA_VIRUS_DEFINITIONS is True:
+        for filename in AV_DEFINITION_EXTRA_FILES:
+            s3_path = os.path.join(AV_DEFINITION_S3_PREFIX, filename)
+            local_path = os.path.join(AV_DEFINITION_PATH, filename)
+            s3_md5 = md5_from_s3_tags(s3_client, bucket, s3_path)
+            if os.path.exists(local_path) and md5_from_file(local_path) == s3_md5:
+                md5_matches.add(filename)
+                continue
+            if s3_md5:
+                to_download[filename] = {
+                    "s3_path": s3_path,
+                    "local_path": local_path,
+                }
+
+    if older_files:
+        print("Not downloading the following older files in series:")
+        print(json.dumps(list(older_files)))
+    if md5_matches:
+        print("Not downloading the following files because local md5 matches s3:")
+        print(json.dumps(list(md5_matches)))
```

Comment:
this is to avoid extremely long logs in cloudwatch, which makes is harder to debug issues